src/ansys/fluent/core/services/interceptors.py

```diff
@@ -29,9 +31,15 @@ def _intercept_call(
         )
         response = continuation(client_call_details, request)
         if not response.exception():
-            network_logger.debug(
-                f"GRPC_TRACE: response = {MessageToDict(response.result())}"
-            )
+            response_str = str(MessageToDict(response.result()))
+            response_bytes = sys.getsizeof(response_str)
+            if response_bytes < log_bytes_limit:
+                network_logger.debug(f"GRPC_TRACE: response = {response_str}")
+            else:
+                network_logger.debug(
```

Comment:
should we hide it or truncate it?

Reply:
Good question.

We could truncate to show the beginning of the response, but I think that is rarely going to be useful unless we are specifically trying to debug gRPC; and in that case we want to see the full response string not a truncated portion anyway, so setting `pyfluent.services.interceptors.log_bytes_limit` to a higher value, as needed, would make more sense to me

We could add an env var to make this easier to control in different environments

We could also make it so `log_bytes_limit` can be manually set to `None` (perhaps through env var definitions) so it does not hide any messages when desired

Reply:
Added optional `PYFLUENT_GRPC_LOG_BYTES_LIMIT` env var to make this easier to control, and made it so setting it to 0 disables message hiding and shows the full response, if required